src/configure/resources/tracePoints.ts

```diff
@@ -0,0 +1,34 @@
+export class TracePoints {
+    // Combined telemetry trace points
+    public static telemetryKey: string = '';
+    public static CurrentFunction: string = 'currentFunction';
+    public static RepoProvider: string = 'repoProvider';
+    public static AzureLoginRequired: string = 'azureLoginRequired';
+    public static Command: string = 'command';
+    public static JourneyId: string = 'journeyId';
+    public static EntryPoint: string = 'entryPoint';
+    public static SourceRepoLocation: string = 'sourceRepoLocation';
+    public static NewOrganization: string = 'newOrganization';
+    public static ChosenTemplate: string = 'chosenTemplate';
```

Comment:
should we also have telemetry for count/list of templates shown?

Reply:
Today we are showing all templates, hence did not think of doing it. Also there is an ask by Pulkit to log the counts of different QuickPickList and the time needed to load them. Hence rather than aiming at each individually, we can solve it generally.
I was thinking of doing that as another iteration, but we can discuss if this needs to be solved now.

Reply:
Update here : We will be showing only simple web app template incase of non-node projects.  This logic has been added in my latest PR.

Reply:
updated and added the list count for:
1. Organization list count
2. Project list count
3. Azure WebApp list count